packages/netlify-cms-core/src/constants/configSchema.js

```diff
@@ -1,8 +1,43 @@
 import AJV from 'ajv';
-import { select, uniqueItemProperties, instanceof as instanceOf } from 'ajv-keywords/keywords';
+import {
+  select,
+  uniqueItemProperties,
+  instanceof as instanceOf,
+  prohibited,
+} from 'ajv-keywords/keywords';
 import ajvErrors from 'ajv-errors';
 import { formatExtensions, frontmatterFormats, extensionFormatters } from 'Formats/formats';
 import { getWidgets } from 'Lib/registry';
+import { I18N_STRUCTURE, I18N_FIELD } from '../lib/i18n';
+
+const localeType = { type: 'string', minLength: 2, maxLength: 10, pattern: '^[a-zA-Z-_]+$' };
```

Comment:
Not sure if there is a error at UI when the locales string is not valid.
But it would be great if there a document to explain the limitation of the locale string.
Because some people like me, may not take a look at the code. 

Example:
**Limitation Locale Strings**
- Minimum length 2
- Maximum length 10
- Allow characters a-z,A-Z and "-", "_"

Reply:
We use https://ajv.js.org/ for validation so a relevant error is printed.
An example error:
![image](https://user-images.githubusercontent.com/26760571/90400133-ae321e80-e0a4-11ea-8274-eaf6ad3925ec.png)